qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -52,7 +52,7 @@
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
  * @since 0.3
- * @checkstyle MultipleStringLiterals (300 lines)
+ * @checkstyle MultipleStringLiterals (400 lines)
```

Comment:
@pinaf I've added some code in the middle of `CheckstyleValidatorTest.java`, so I needed to extend suppression a bit